src/test/java/org/reaktivity/nukleus/tcp/internal/streams/rfc793/ServerIOExceptionFromReadIT.java

```diff
@@ -87,7 +87,7 @@ public void shouldReportIOExceptionFromReadAsAbortAndReset() throws Exception
         "${route}/server/controller",
         "${server}/server.received.abort.sent.end/server"
     })
-    public void shouldNotResetWhenProcessingEndAfterIOExceptionFromRead() throws Exception
+    public void shouldNotAbortWhenProcessingEndAfterIOExceptionFromRead() throws Exception
```

Comment:
Please restore the original name, for the same reasons as in my comment above on ClientIOExceptionFromReadIT.

Reply:
Done.